charts/etcd/templates/etcd-statefulset.yaml

```diff
@@ -307,6 +307,10 @@ spec:
               optional: true
 {{- end }}
         volumeMounts:
+{{- if eq .Values.store.storageProvider "Local" }}
+        - name: host-storage
+          mountPath: {{ .Values.store.storageContainer }}
+{{- end }}
```

Comment:
so, you are mounting a volume for `Local` provider for data-storage ... right ?

Reply:
Yes, exactly. That directory will be used to write the backups there